portfolio/src/main/webapp/index.html

```diff
@@ -29,8 +29,8 @@
 
     <div id="content">
       <div class="titlecontainer">
-        <span class="homepageheader1">Andrew Gan</span>
-        <span class="homepageheader2">Sit Back & Relax!</span>
+        <span class="homepage-header-1">Andrew Gan</span>
+        <span class="homepage-header-2">Sit Back & Relax!</span>
```

Comment:
This is a good change but puzzled as to why it's in this PR? It would be best to have it in a dedicated CSS cleanup PR.